src/services/notifications.service.js

```diff
@@ -191,35 +174,52 @@ const sendNotifications = async (
   lcContactConfig,
   registration,
   postRegistrationMetadata,
-  notifyTemplateKeys
+  configData
 ) => {
+  let emailsToSend = [];
+
   for (let typeOfCouncil in lcContactConfig) {
     const lcNotificationEmailAddresses =
       lcContactConfig[typeOfCouncil].local_council_notify_emails;
 
     for (let recipientEmailAddress in lcNotificationEmailAddresses) {
-      await sendEmailOfType(
-        "LC",
-        registration,
-        postRegistrationMetadata,
-        lcContactConfig,
-        lcNotificationEmailAddresses[recipientEmailAddress],
-        notifyTemplateKeys
-      );
+      emailsToSend.push({
+        type: "LC",
+        address: lcNotificationEmailAddresses[recipientEmailAddress],
+        templateId: configData.notify_template_keys.lc_new_registration
+      });
     }
   }
 
   const fboEmailAddress =
     registration.establishment.operator.operator_email ||
     registration.establishment.operator.contact_representative_email;
 
-  await sendEmailOfType(
-    "FBO",
+  emailsToSend.push({
+    type: "FBO",
+    address: fboEmailAddress,
+    templateId: configData.notify_template_keys.fbo_submission_complete
+  });
+
+  if (registration.metadata.feedback1) {
```

Comment:
There is no unit test for this path

Reply:
Unit test wasn't running due to incorrect beforeEach set up, those tests now running